notebooks/social_vulnerability.ipynb

```diff
@@ -75,27 +75,27 @@
    "source": [
     "def demographic_factors(state_number, county_number, year, geo_type=\"tract:*\"):\n",
     "\n",
-    "    _, df_1 = CensusUtil.get_census_data(state=state_code, county=county_code, year=year,\n",
+    "    _, df_1, _ = CensusUtil.get_census_data(state=state_code, county=county_code, year=year,\n",
```

Comment:
I looked at the pyincore-data-example and I wonder if it's worth giving those variables more meaningful names so it's clear what they are, even if we aren't using them. Something like:

api_json, df, dataset = CensusUtil.get_census_data...

Reply:
@Rashmil-1999 Can you update the variable naming?

Reply:
Also, can you fix the changelog conflict?

Reply:
Okay, got it. I used the throw-away variables as I thought we don't require others. I will fix it soon.

Reply:
We don't right now, but I think it's just clearer what is being returned from the method and in case we have a future need to use the dataset returned. Thanks for updating it.